presto-product-tests/src/main/java/com/facebook/presto/tests/hive/TestSqlStandardAccessControlChecks.java

```diff
@@ -0,0 +1,175 @@
+    }
+
+    @Test(groups = {AUTHORIZATION, HIVE_CONNECTOR, PROFILE_SPECIFIC_TESTS})
+    public void testAccessControlDropTable()
+    {
+        String tableName = "alice_table_for_drop";
+        QueryExecutor queryExecutorForAlice = connectToPresto("alice@presto");
+        QueryExecutor queryExecutorForBob = connectToPresto("bob@presto");
+
+        queryExecutorForAlice.executeQuery(format("CREATE TABLE %s(month bigint, day bigint)", tableName));
+
+        assertThat(() -> queryExecutorForBob.executeQuery(format("DROP TABLE %s", tableName))).
+                failsWithMessage(format("Access Denied: Cannot drop table default.%s", tableName));
+        queryExecutorForAlice.executeQuery(format("DROP TABLE %s", tableName));
+        assertThat(() -> queryExecutorForAlice.executeQuery(format("SELECT * FROM %s", tableName)))
+                .failsWithMessage("does not exist");
+    }
+
+    @Test(groups = {AUTHORIZATION, HIVE_CONNECTOR, PROFILE_SPECIFIC_TESTS})
+    public void testAccessControlAlterTable()
+    {
+        String tableName = "alice_owned_table";
+        QueryExecutor queryExecutorForAlice = connectToPresto("alice@presto");
+        QueryExecutor queryExecutorForBob = connectToPresto("bob@presto");
+
+        queryExecutorForAlice.executeQuery(format("DROP TABLE IF EXISTS %s", tableName));
+        queryExecutorForAlice.executeQuery(format("CREATE TABLE %s(month bigint, day bigint)", tableName));
+
+        assertThat(() -> queryExecutorForBob.executeQuery(format("ALTER TABLE %s ADD COLUMN year bigint", tableName))).
+                failsWithMessage(format("Access Denied: Cannot add a column to table default.%s", tableName));
+        queryExecutorForAlice.executeQuery(format("ALTER TABLE %s ADD COLUMN year bigint", tableName));
+    }
+
+    @Test(groups = {AUTHORIZATION, HIVE_CONNECTOR, PROFILE_SPECIFIC_TESTS})
+    public void testAccessControlCreateView()
+    {
+        String tableName = "alice_owned_table";
+        String viewName = "bob_view";
+        QueryExecutor queryExecutorForAlice = connectToPresto("alice@presto");
+        QueryExecutor queryExecutorForBob = connectToPresto("bob@presto");
+
+        queryExecutorForAlice.executeQuery(format("DROP TABLE IF EXISTS %s", tableName));
+        queryExecutorForAlice.executeQuery(format("CREATE TABLE %s(month bigint, day bigint)", tableName));
+
+        queryExecutorForBob.executeQuery(format("DROP VIEW IF EXISTS %s", viewName));
+        assertThat(() -> queryExecutorForBob.executeQuery(format("CREATE VIEW %s AS select * from %s", viewName, tableName))).
+                failsWithMessage(format("Access Denied: Cannot select from table default.%s", tableName));
+        queryExecutorForAlice.executeQuery(format("GRANT SELECT ON %s TO bob", tableName));
+        assertThat(() -> queryExecutorForBob.executeQuery(format("CREATE VIEW %s AS select * from %s", viewName, tableName))).
```

Comment:
Use `createViewAs()` here and above. Moreover I would extract this view creation sql as variable so reader would not immediately that you are creating the same view each time